api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -301,6 +301,46 @@ message HttpConnectionManager {
     type.http.v3.PathTransformation http_filter_transformation = 2;
   }
 
+  // Configures the manner in which the Proxy-Status HTTP response header is
+  // populated.
+  //
+  // See the Proxy-Status RFC: https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-proxy-status-05
+  // TODO(ambuc): Update this with the non-draft URL when finalized.
+  //
+  // The Proxy-Status header is a string of the form:
+  //
+  //   "<server_name>; error=<error_type>; details=<details>"
+  message ProxyStatusConfig {
+    enum ProxyName {
+      // Use the string "envoy".
+      ENVOY_LITERAL = 0;
+
+      // Use the name of the server as it appears in
+      // HttpConnectionManager.server_name.
+      SERVER_NAME = 1;
+    }
+
+    // If true, attaches the Proxy-Status header to response headers.
+    // This value defaults to `false`, i.e. no Proxy-Status header is attached.
+    bool attach_proxy_status = 1;
```

Comment:
Is there a use case for having the proxy_status_config and not sending the header? If not I'd nix this boolean
If we keep it, maybe send_proxy_status rather than attach?

Reply:
I agree, this is superfluous since you already have a message to indicate that Proxy-Status should be enabled.